athena-federation-integ-test/README.md

```diff
@@ -88,7 +88,35 @@ Provide implementation for the following 4 abstract methods in the test class:
      * access to multiple connector-specific AWS services (e.g. DynamoDB, Elasticsearch etc...)
      * @return A policy document object.
      */
-    protected abstract PolicyDocument getConnectorAccessPolicy();
+    protected abstract Optional<PolicyDocument> getConnectorAccessPolicy();
+```
+
+### VPC Configuration
+
+The Integration-Test module can be adapted to test connectors that utilize a VPC configuration to connect to the data
+source. The framework can detect the default VPC configuration set in the specified AWS account and use it to configure
+the Lambda function. In order for the latter to be able to connect to the data source, however, the same VPC configuration
+must be set when provisioning the DB instance.
+
+To use a VPC configuration, the following method must be overridden in your integration test class to `return true`
+(default is false):
+
+```java
+    /**
+     * Must be overridden in the extending class when the connector supports a VPC configuration.
+     * @return false (default)
+     */
+    protected boolean connectorSupportsVpcConfig()
```

Comment:
What is an example of a connector that doesn't support VPC config?

Reply:
DynamoDB